R/generate_simulated_data.R

```diff
@@ -143,11 +143,7 @@ generate_simulated_data <- function(r_in_weeks = # nolint
   model$expose_functions(global = TRUE)
 
   # Pull parameter values into memory
-  params <- get_params(input_params_path) # load in a single row tibble
-  par_names <- colnames(params) # pull them into memory
-  for (i in seq_along(par_names)) {
-    assign(par_names[i], as.double(params[i]))
-  }
+  params <- get_params(input_params_path)
```

Comment:
I can see why you maybe want to do this but it doesn't seem like it belongs in this PR

Reply:
For a future refactor I think its quite clunnky to get_params internally like this. I would either do it in the default settings of the function (i.e get_params(path)) or (much more likely) do it using fn args to give more idiomatic R code and then have a pattern where you use do.call externally if wanting to run against a param list

Reply:
I agree but a minimal level of restructure was necessary so that vector-valued parameters ($\alpha$ vector of the Dirichlet prior) could be passed). And this accords with what is happening in #29

Reply:
hmm okedokie